src/platforms/flutter/upload-debug.mdx

```diff
@@ -18,32 +18,136 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
+
+`upload_debug_symbols`
+
+: Enables or disables the automatic upload of debug symbols. This is a boolean type with default value `true`.
+
+`upload_source_maps`
+
+: Enables or disables the automatic upload of source maps. This is a boolean type with default value `false`.
+
+`upload_sources`
+
+: Does or doesn't include the source code of native code. This is a boolean type with default value `false`.
+
+`url`
+
+: The URL of your project e.g. `https<area>://mysentry.invalid/`. This is a `string` type. The alternative environmental variable is `SENTRY_URL`.
+
+`wait_for_processing`
+
+: Wait for server-side processing of uploaded files. This is a boolean type with default value `false`.
+
+`log_level`
+
+: Configures the log level for sentry-cli. This is a `string` type with default value `warn`. The alternative environmental variable is `SENTRY_LOG_LEVEL`.
+
+`release`
+
+: The release version for source maps, it should match the release set by the SDK. This is a `string` type with default value `name@version` from pubspec. The alternative environmental variable is `SENTRY_RELEASE`.
+
+`web_build_path`
+
+: The web build folder. This is a `string` type with default value `build/web`.
+
+`commits`
+
+: Release commits integration. This is a `string` type with default value `auto`.
+
+`ignore_missing`
+
+: Ignore missing commits previously used in the release. This is a boolean type with default value `false`.
+
+### Run
+
+```bash
+flutter packages pub run sentry_dart_plugin
+```
+
+### Troubleshooting
+
+Refer to [Troubleshooting - Sentry Dart Plugin](/platforms/flutter/troubleshooting#sentry-dart-plugin) to resolve potential issues.
+
+## Manually Upload Debug Symbols
+
+### Uploading for iOS and macOS
 
 Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. [Learn how to upload the dSYM files](/platforms/apple/dsym/).
 
 If you use the `split-debug-info` and `obfuscate` flags, you need to upload the `*.dSYM` files instead of the `*.symbols` files generated by the Flutter build. The `*.dSYM` files are located in the `build` folder of your project and not the given `split-debug-info` folder.
 
-## Uploading for Android NDK
+### Uploading for Android NDK
+
+The NDK symbols are uploaded by the Sentry Dart Plugin as well if `upload_debug_symbols` option is enabled.
 
-If you are using `sentry_flutter` version earlier than 5.1, native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` for [Android App Bundle](https://developer.android.com/guide/app-bundle) in your `gradle.properties` file.
+Check out [Uploading Files](/product/cli/dif/#uploading-files) if you want to upload manually directly with sentry-cli.
```

Comment:
Let's agree on the fallback phrase for the CLI.

This one is:

> Check out [Uploading Files](/product/cli/dif/#uploading-files) if you want to upload manually directly with sentry-cli.

The other one is:

> . As an alternative, you can also use our CLI to [upload source maps for Flutter Web](/product/cli/releases/#managing-release-artifacts).

Both are fine, let's just stick to one of them, the later is nicer IMO.